### PR TITLE
Attempt to eliminate security-compromised dependencies

### PR DESCRIPTION
**Pull request by waxlamp**
This PR attempts to upgrade all dependencies to recent versions in order to eliminate some or all of the security alerts raised [here](https://github.com/ToyotaResearchInstitute/materialnet/network/alerts). I am not sure if I succeeded in full.

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -9,8 +9,7 @@ import datasets from '../datasets';
 import { isEqual, camelCase, uniqueId } from 'lodash-es';
 import { createFormatter } from './format';
 
-// eslint-disable-next-line import/no-webpack-loader-syntax
-import Worker from 'worker-loader!../worker';
+import Worker from '../layout.worker.js';
```
Comment (alesgenova) [inline src/store/index.js:12]: This import cannot be resolved.
I'm assuming it should be
`import {layout as Worker} from '../worker';`

and export the `layout` function from the `src/worker/index.js`.

However if that is what should be imported, it takes 3 parameters when newing, but in this file it's instantiated as `new Worker()`, leading to runtime errors

Comment (waxlamp) [reply src/store/index.js:12]: Oops, it looks like I forgot to properly track a moved file. Can you try again after updating to 
b37a958?

Comment (alesgenova) [reply src/store/index.js:12]: It's working well now!
One thing I'd change is removing the `.js` extension from the import here, and in a couple of other places

Comment (waxlamp) [reply src/store/index.js]: Good suggestion; see d5d46e7d6036469cc2d81c19c0411d1c614dd74a.

**Pull request merged**